infra.bs

```diff
@@ -221,47 +223,51 @@ in parentheses. [[!UNICODE]]
 <div class=example id=example-code-point-notation>
  <p>The <a>code point</a> rendered as 🤔 is represented as U+1F914.
 
- <p>When referring to that <a>code point</a>, we might instead say "U+1F914 THINKING FACE (🤔)",
- instead of just "U+1F914", to provide extra context.
+ <p>When referring to that <a>code point</a>, we might say "U+1F914 (🤔)", to provide extra context.
+
+ <p><a>Code points</a> that are difficult to render unambigiously, such as U+000A, can be referred
+ to as "U+000A LINE FEED".
 </div>
 
 <p>In certain contexts <a>code points</a> are prefixed with "0x" instead of "U+".
 
 <p>A <dfn export>scalar value</dfn> is a <a>code point</a> that is not in the range
 U+D800 to U+DFFF, inclusive.
 
-<p>An <dfn export>ASCII code point</dfn> is a <a>code point</a> in the range U+0000 to U+007F,
-inclusive.
+<p>An <dfn export>ASCII code point</dfn> is a <a>code point</a> in the range U+0000 NULL to
+U+007F DELETE, inclusive.
 
 <p>An <dfn export lt="ASCII tab or newline|ASCII tabs or newlines">ASCII tab or newline</dfn> is
-U+0009, U+000A, or U+000D.
+U+0009 CHARACTER TABULATION, U+000A LF, or U+000D CR.
 
-<p>An <dfn export>ASCII whitespace</dfn> is U+0009, U+000A, U+000C, U+000D, or U+0020.
+<p>An <dfn export>ASCII whitespace</dfn> is U+0009 CHARACTER TABULATION, U+000A LF, U+000C FF,
+U+000D CR, or U+0020 SPACE.
 
 <p class=note>"Whitespace" is a mass noun.
 
-<p>A <dfn export>C0 control</dfn> is a <a>code point</a> in the range U+0000 to U+001F, inclusive.
+<p>A <dfn export>C0 control</dfn> is a <a>code point</a> in the range U+0000 NULL to
+U+001F INFORMATION SEPARATOR ONE, inclusive.
 
 <p>A <dfn export lt="C0 control or space|C0 controls or spaces">C0 control or space</dfn> is a
-<a>C0 control</a> or U+0020.
+<a>C0 control</a> or U+0020 SPACE.
 
-<p>An <dfn export>ASCII digit</dfn> is a <a>code point</a> in the range U+0030 to U+0039,
+<p>An <dfn export>ASCII digit</dfn> is a <a>code point</a> in the range U+0030 (0) to U+0039 (9),
 inclusive.
 
 <p>An <dfn export>ASCII upper hex digit</dfn> is an <a>ASCII digit</a> or a <a>code point</a> in the
-range U+0041 to U+0046, inclusive.
+range U+0041 (A) to U+0046 (F), inclusive.
 
 <p>An <dfn export>ASCII lower hex digit</dfn> is an <a>ASCII digit</a> or a <a>code point</a> in the
-range U+0061 to U+0066, inclusive.
+range U+0061 (a) to U+0066 (f), inclusive.
 
 <p>An <dfn export>ASCII hex digit</dfn> is an <a>ASCII upper hex digit</a> or
 <a>ASCII lower hex digit</a>.
 
-<p>An <dfn export>ASCII upper alpha</dfn> is a <a>code point</a> in the range U+0041 to U+005A,
-inclusive.
+<p>An <dfn export>ASCII upper alpha</dfn> is a <a>code point</a> in the range U+0041 (A) to
```

Comment:
Oh, I missed that this was inside the parenthesis, so we don't use the names there.